addons/quickmount/functions/fnc_getInNearest.sqf

```diff
@@ -0,0 +1,95 @@
+/*
+ * Author: Kingsley
+ * Mount the player in the vehicle they are directly looking at based on their distance.
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [] call ace_quickmount_fnc_getInNearest;
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+if (!GVAR(enabled) ||
+    {isNull ACE_player} ||
+    {vehicle ACE_player != ACE_player} ||
+    {!alive ACE_player} ||
+    {ACE_player getVariable ["ace_unconscious", false]}
+) exitWith {};
+
+private _start = AGLtoASL positionCameraToWorld [0, 0, 0];
```

Comment:
`private _start = AGLtoASL (ACE_player modelToWorldVisual (ACE_player selectionPosition "pilot"));`
I think this will work a lot better for people using 3rd person